style.css

```diff
@@ -1,21 +1,36 @@
 body {
   scroll-behavior: smooth;
-  background-color: rgba(228, 218, 218, 0.36);
   margin: 0;
-  padding: 0%;
-  width: auto;
+  width: 100vh;
 }
 
```

Comment:
- [ ] [REQUIRED] - Good job working on this project, I know that this design is a big challenge. However after you made the changes, it's exceeding the borders. It seems that there is a problem related to the width. Please, fix that.

How it looks like:
![image](https://user-images.githubusercontent.com/61767582/176800547-1915043c-f8d6-4946-b59d-3a5c98ba8775.png)

Reply:
@leopiresgoss Hi leo! I have a question, what are the dimensions that we should use for the mobile version? Adjust my program so that below 500px will fit the mobile version.

Below I share the section of the code that controls it.

**Screenshot of My Portfolio:**
![Mi Portfolio](https://user-images.githubusercontent.com/106561762/176802484-9359d608-9917-4856-be78-3f7759759dad.png)

**Sreenshot of the code about the constrol.:**
![Code mobile version](https://user-images.githubusercontent.com/106561762/176802772-0385a6a9-5ea8-4be3-850a-70b2b211def0.png)

**If you have a moment, I'd like to have a quick Zoom call, please.**

Reply:
@leopiresgoss I was able to fix it! Some images were misconfigured and that is why the program sometimes looked weird.

Thank you for your time. Have a great day!